utils.py

```diff
@@ -31,12 +29,19 @@ def take_screenshot():
 
 
 def prepare_image(img):
+    import wx
+    wx.App()
```

Comment:
Alright, this has been updated. I had to leave the import since we are checking for a `wx` type in the `prepare_image` call, but didn't need to call `wx.App()` anymore. And `take_screenshot` has been moved to `record.py` and removed from `utils.py`. Recording and preparing still worked as expected.